ui/src/components/Common/MediaCard/MediaModal/index.tsx

```diff
@@ -0,0 +1,283 @@
+                          audience: 'icons_logos/rt_audience.svg',
+                        },
+                        themoviedb: {
+                          audience: '/icons_logos/tmdb_icon.svg',
+                        },
+                      }
+
+                      const icon = iconMap[prefix]?.[type]
+                      return (
+                        <div
+                          key={index}
+                          className="mb-1 flex items-center justify-between space-x-1 rounded-lg bg-black bg-opacity-70 px-3 py-1 text-white shadow-lg"
+                        >
+                          <img
+                            src={icon}
+                            alt={`${prefix} ${type} Icon`}
+                            className="h-6 w-6"
+                            title={`${prefix}-${type}`}
+                          />
+                          <span className="flex cursor-default items-center justify-end text-sm font-medium">
+                            {rating.value}
+                          </span>
+                        </div>
+                      )
+                    })}
+                  </div>
+                ) : (
+                  ''
+                )}
+                {metadata.Genre && metadata.Genre.length > 0 ? (
+                  <div className="pointer-events-none absolute bottom-4 right-4 z-10 flex flex-wrap">
+                    {metadata.Genre.map((genre, index) => (
+                      <span
+                        key={index}
+                        className="mr-0.5 flex h-8 items-center justify-center rounded-lg bg-black bg-opacity-70 px-2 font-medium text-white shadow-lg"
+                      >
+                        {genre.tag}
+                      </span>
+                    ))}
+                  </div>
+                ) : (
+                  ''
+                )}
+              </>
+            )}
+          </div>
+
+          <div className="p-4">
+            <div className="flex items-center justify-between border-b pb-4">
+              <div>
+                <h2 className="text-xl font-semibold text-gray-100">
+                  {title} ({year})
+                </h2>
+              </div>
+            </div>
+
+            <div className="mt-2 text-gray-300">
+              <p>{summary || 'No summary available.'}</p>
+            </div>
+
+            <div className="mr-0.5 mt-6 flex flex-row items-center justify-between">
+              {metadata?.Guid &&
+                ['movie', 'show'].includes(mediaType) &&
+                metadata.Guid.length > 0 && (
+                  <div className="flex flex-wrap items-center text-xs text-zinc-400">
+                    {metadata.Guid.map((guid, index) => (
+                      <span
+                        key={index}
+                        className="mb-0.5 mr-0.5 flex h-8 items-center justify-center rounded-lg bg-zinc-700 bg-opacity-70 px-2 text-xs text-white shadow-lg"
```

Comment:
I think it's cleaner to use gap on the parent for margins

```suggestion
                        className="flex h-8 items-center justify-center rounded-lg bg-zinc-700 bg-opacity-70 px-2 text-xs text-white shadow-lg"
```